extensions/import/src/wizard/fileConfig.ts

```diff
@@ -2,15 +2,126 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
+
 'use strict';
 
 import * as vscode from 'vscode';
-import * as path from 'path';
 import * as sqlops from 'sqlops';
 
-export async function fileConfig(view: sqlops.ModelView) : Promise<void> {
-	//from services sample placeholder code
-	let formWrapper = view.modelBuilder.loadingComponent().component();
+let server: sqlops.connection.Connection = null;
```

Comment:
we prefer `undefined` to `null`. If you just write `let server: sqlops.connection.Connection;` then it will get initialized as undefined